src/integration-test/java/developerTC/TestAgentRegistration_04.java

```diff
@@ -1,84 +1,66 @@
 import sg.dex.starfish.util.DID;
 import sg.dex.starfish.util.JSON;
 import sg.dex.starfish.util.Utils;
 
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
 /**
  * As a developer building or managing an Ocean Agent,
  * I need to be able to register my Agent on the network and obtain an Agent ID
  */
 public class TestAgentRegistration_04 {
-    @Test
-    public void testRegistration() {
-        Map<String, Object> ddo = new HashMap<>();
-        List<Map<String, Object>> services = new ArrayList<>();
-        services.add(Utils.mapOf(
-                "type", "Ocean.Meta.v1",
-                "serviceEndpoint", "/api/v1/meta"));
-        services.add(Utils.mapOf(
-                "type", "Ocean.Storage.v1",
-                "serviceEndpoint", "/api/v1/assets"));
-        services.add(Utils.mapOf(
-                "type", "Ocean.Invoke.v1",
-                "serviceEndpoint", "/api/v1/invoke"));
-        services.add(Utils.mapOf(
-                "type", "Ocean.Auth.v1",
-                "serviceEndpoint", "/api/v1/auth"));
-        services.add(Utils.mapOf(
-                "type", "Ocean.Market.v1",
-                "serviceEndpoint", "/api/v1/market"));
-        ddo.put("service", services);
-        String ddoString = JSON.toPrettyString(ddo);
-
-        Resolver resolver= new LocalResolverImpl();
-        // creating unique DID
-        DID surferDID = DID.createRandom();
-        //registering the  DID and DDO
-        resolver.registerDID(surferDID, ddoString);
-
-        // creating a Remote agent instance for given Ocean and DID
-        RemoteAgent remoteAgent = RemoteAgent.create(resolver, surferDID);
-        assumeTrue(null != remoteAgent);
-        assertEquals(remoteAgent.getDID(), surferDID);
-        // verify the DID format
-        assertEquals(remoteAgent.getDID().getMethod(), "op");
-        assertEquals(remoteAgent.getDID().getScheme(), "did");
-        assumeTrue(null != remoteAgent.getDDO());
-    }
+
 
     @Test
-    public void testRegistrationForException() {
-        Map<String, Object> ddo = new HashMap<>();
+    public void testRegistration() throws IOException {
 
-        String ddoString = JSON.toPrettyString(ddo);
 
-        //Should not allow to create the null DID ?
-        //getting the default Ocean instance
-        Resolver resolver=new LocalResolverImpl();
-        RemoteAgent remoteAgent = RemoteAgent.create(resolver, null);
-        //registering the  DID and DDO
+        Resolver resolver=DexResolver.create();
+        DID did = DID.createRandom();
+        resolver.registerDID(did,getAgentDDO());
 
-        assertThrows(IllegalArgumentException.class, () -> {
-            resolver.registerDID(null, ddoString);
-        });
 
```

Comment:
Need to test that getting the DDO for the registered DID returns exactly the same DDO String